### ISSUE TITLE
Is this ready for Avalonia 11? Showing RC in an open PR

### ISSUE THREAD

**Issue by Geektoolkit** [AvaloniaUI/Avalonia.GIF#31]
I'm looking to integrate this into my current codebase as I update to Avalonia 11. Is the open PR the right branch to pull or can we please get that integrated so I can test it out? This is a big feature for the users fo Dynaframe :)

**Comment by xafero** [comment]
I updated the 11.rc PR branch to the current release version and it seems to work (it has conflicts with main branch, so no auto merge available):
https://github.com/AvaloniaUI/Avalonia.GIF/pull/32

**Comment by Geektoolkit** [comment]
Yeah that's what I'm shipping to my customers now. Just feels awkward to ship out of an RC branch

**Comment by jmacato** [comment]
Hi @Geektoolkit ! Long time no chat!

I've just updated this repo and we have plans to release this as a nuget soon!
